Framework/Kernel/src/UsageService.cpp

```diff
@@ -273,9 +273,6 @@ std::string UsageServiceImpl::generateStartupMessage() {
   message["osVersion"] = ConfigService::Instance().getOSVersion();
   message["osReadable"] = ConfigService::Instance().getOSVersionReadable();
 
-  // legacy interface requires paraview version
-  message["ParaView"] = 0;
```

Comment:
This cannot go. I don't believe the webapi knows what to do when this key is missing

Reply:
Thank you for letting me know. I will add it back :+1: